README.md

```diff
@@ -52,6 +52,8 @@ OPTIONS:
 ### Packages
  * [Releases](https://github.com/Umio-Yasuno/amdgpu_top/releases)
     * .deb, .AppImage
+ * [AUR](https://aur.archlinux.org/packages/amdgpu_top)
+    * made by Teddy-Kun
```

Comment:
Sorry, do we need the name of the package producer in the README?  
The README.md file should contain the necessary information.

Reply:
I personally don't think so, but some people value knowing that it is not made by the owner

Reply:
Assuming there are such people, I do not wish to make an appeal to them.

Reply:
Ok, its removed